pkgs/applications/window-managers/fvwm/default.nix

```
@@ -1,28 +1,46 @@
-{ gestures ? false
-, lib, stdenv, fetchurl, pkg-config
-, cairo, fontconfig, freetype, libXft, libXcursor, libXinerama
-, libXpm, libXt, librsvg, libpng, fribidi, perl
-, libstroke ? null
-}:
+{ autoconf, automake, gestures ? false, lib, stdenv, fetchFromGitHub, pkg-config
```

```suggestion
{ autoconf, automake, enableGestures ? false, lib, stdenv, fetchFromGitHub, pkg-config
```

that prevents this from breaking when there is a gestures package.

---

Done.